src/doc/topics/reusable-api.md

```diff
@@ -0,0 +1,53 @@
+## The Reusable Chart API
+
+The Reusable Chart API is a modular structure to create and reuse D3.js components. We saw the first example of this pattern in Mike Bostock’s seminal post [Towards Reusable Charts (2012)][towardsreusablecharts].
+
+We built Britecharts using this design pattern, so that all our components benefit from its characteristics. They are all _simple, modular, reusable, composable, and testable_.
+
+The pattern is based in a closure that allows us to keep properties private and define the elements of the components' API. Take a look at the code:
```

Comment:
"This pattern resides within a closure that.." would make more sense here but it's your call!